ui/app/components/job-status/allocation-row.hbs

```diff
@@ -0,0 +1,22 @@
+{{#if this.showSummaries}}
+  <div class="alloc-status-summaries"
+    {{did-insert this.captureElement}}
+    {{window-resize this.reflow}}
+  >
+    {{#each-in @allocBlocks as |status allocs|}}
+      {{#if (gt allocs.length 0)}}
+        <JobStatus::AllocationStatusBlock @status={{status}} @count={{allocs.length}} @width={{compute (action this.calcPerc) allocs.length}} />
+      {{/if}}
+    {{/each-in}}
+  </div>
+{{else}}
+  <div class="ungrouped-allocs">
+    {{#each-in @allocBlocks as |status allocs|}}
+      {{#if (gt allocs.length 0)}}
+        {{#each (range 0 allocs.length)}}
+          <span class="represented-allocation {{status}}"></span>
+        {{/each}}
+      {{/if}}
+    {{/each-in}}
+  </div>
+{{/if}}
```

Comment:
Might rename this to something other than allocation-row for ambiguity's sake. Open to suggetsions!

Reply:
Ended up renaming to allocation-status-row!